### PR TITLE
Minor typo improvements

### PR DESCRIPTION
**Pull request by vadi2**
I didn't re-order buttongroups, Qt Creator did that upon saving.

### CODE REVIEW

File: src/dlgRoomExits.cpp [hunk 1]
```diff
@@ -1571,7 +1571,7 @@ void dlgRoomExits::init( int id ) {
 
     roomID->setText(QString::number(id));
     roomWeight->setText(QString::number(pR->getWeight()));
-    QString titleText = "Exits for Room";
```
Comment (SlySven) [inline src/dlgRoomExits.cpp:1574]: Only thing I'd say about this is that here we are referring to a definite article, the Room, in question so Room is not so unreasonable I think, note also that for a named/unnamed room the results would be something like:
Exits for Room "The Tar Valon Road" / Exits for Room Id 12345
Compare this to:
Exits for room "The Tar Valon Road" / Exits for room Id 12345
Admittedly the Mud I tend to play capitalises non-trivial words in room names but for this usage case I would defend 'R' and after all this is the dialogue's "Title"... 8-)

Comment (vadi2) [inline src/dlgRoomExits.cpp:1574]: It's just a room, not the Almighty Room of Doom though... having many Rooms kind of takes away from the purpose, no?

It still reads weird to me either way.

**Comment by SlySven** [comment]
Yeah, I'd found that Qt Creator / Designer (plug-in) does rearrange when saving a form sometimes reordering things in grid-layouts.  I've had to use the plain text editor to reorder elements before now, though the indentation does help with that.

I don't know whether the Qt guys ever did sort out setting radio buttons' id numbers programmatically - you may find one of two sarky comments about that!

Perhaps combo-boxes wouldn't be such a bad idea after all...

**Comment by vadi2** [comment]
"Perhaps combo-boxes wouldn't be such a bad idea after all..." - OK, I'll
redo the rest of the UI then to comboboxes, send you a pull request - would
you be willing to make the backend work with them or should I?

**Comment by SlySven** [comment]
I'll let you take a bash at the back end stuff - I've been clobbering other peoples handiwork for a bit now so what's source of the goose is source for the gander and I need to let others go over what I've munged together.  I recommend doing just the n exit to check out the functionality - that is generally the second one in my coding (the nw one is the one with all the comments), watch out for the enabling and disabling as other controls are altered (and forced resetting to the "no door" state).

**Pull request closed without merging**